### PR TITLE
cp: Reduce the size of tarball to continue ASF release (#15004)

### PR DESCRIPTION
**Pull request by zhongjiajie**
* Reduce the size of tarball to continue ASF release

for more detail you can see https://lists.apache.org/thread/rmp7fghlj0n7h9y2v3p8gkw9f9qbo6qt

### CODE REVIEW

File: tools/dependencies/known-dependencies.txt [hunk 1]
```diff
@@ -435,8 +432,8 @@ azure-core-management-1.10.1.jar
 api-common-2.6.0.jar
 auto-value-1.10.1.jar
 auto-value-annotations-1.10.1.jar
-bcpkix-jdk15on-1.67.jar
-bcprov-jdk15on-1.67.jar
+bcpkix-jdk15on-1.69.jar
+bcprov-jdk15on-1.69.jar
```
Comment (ruanwenjun) [inline tools/dependencies/known-dependencies.txt:436]: Doesn't this will affect the license ?

**Review by ruanwenjun — COMMENTED** [review]
CI failed, this PR shouldn't affect ci.

File: tools/dependencies/known-dependencies.txt (hunk 1 above)
Comment (zhongjiajie) [reply tools/dependencies/known-dependencies.txt:436]: will try to fix it tomorrow or later

**Comment by zhongjiajie** [comment]
ci pass, and I would like to merge this to prepare branch

**Pull request merged**